chat.js

```diff
@@ -0,0 +1 @@
+//About to build chat
```

Comment:
@alitsdev Please add a space between comment.

Reply:
I´m on it

Reply:
👍